pages/dkp/konvoy/2.2/choose-infrastructure/aws/advanced/permissions_role_create_cluster/index.md

```diff
@@ -0,0 +1,240 @@
+---
+layout: layout.pug
+navigationTitle: Minimal Permissions and Role to Create Clusters
+title: Minimal Permissions and Role to Create Clusters
+excerpt: Minimal Permissions and Role to Create Clusters
+beta: false
+enterprise: false
+menuWeight: 35
+---
+
+Configure IAM Prerequisites before starting a cluster
```

Comment:
Should this be bolded like it is in confluence

Reply:
Yes